### PR TITLE
Student profile

### PR DESCRIPTION
**Pull request by palak199**
This PR adds student profile page. However I have not added functionality of update profile and change password. It solves issue #54 .
@garg3133 please review.

### CODE REVIEW

File: templates/students/profile.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+	</style>
+{% endblock %}
+
+{% block content %}
+
+<div class="container">
+	<div class="py-3 mt-3 text-center">
+		<h2>
+			 Your Profile
```
Comment (garg3133) [inline templates/students/profile.html:23]: Change this to `Student Profile`.

File: templates/students/profile.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+						</div>
+						<div class="col-6 col-lg-8">
+							{{ profile.guardian_name}}
+						</div>
+					</div>
+				</div>
+			</div>
+			<div class="row justify-content-center pt-3">
+				<a href="{% url 'students:update_profile' %}" class="btn btn-info mx-3 my-2">Edit Profile</a> 
+				<a href="{% url 'accounts:password_change' %}" class="btn btn-info mx-3 my-2">Change Password</a>
```
Comment (garg3133) [inline templates/students/profile.html:94]: Students have no passwords. So, you may remove this button.

File: templates/students/profile.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+			</div>
+				
+			{% comment %}
```
Comment (garg3133) [inline templates/students/profile.html:97]: Remove this whole chunk of code. It was just some experimental work in Volunteer Profile.

**Review by garg3133 — COMMENTED** [review]
Keep the order as Name, Class, Village, Gender, Guardian Name, Contact No.

Below are some more required changes. Rest, everything looks good!

**Comment by garg3133** [comment]
Also, there's some issue with `Gender` on small screens. Please check!

Screenshot from 2021-01-22 22-31-27

File: templates/students/profile.html (hunk 1 above)
Comment (palak199) [reply templates/students/profile.html]: I have changed it to Student profile.

File: templates/students/profile.html (hunk 2 above)
Comment (palak199) [reply templates/students/profile.html]: Ok, I removed it.

File: templates/students/profile.html (hunk 3 above)
Comment (palak199) [reply templates/students/profile.html]: sure. There were some CSS classes missing in 'gender' column and now its fine.
image

**Comment by palak199** [comment]
@garg3133 Thanks for reviewing, I have made the required changes.

**Comment by garg3133** [comment]
Looks good! Merging this into master.

Thank you @palak199 for your contribution 🚀

**Pull request merged**